app/components/QrCodeScanner/QrCodeScanner.jsx

```diff
@@ -46,7 +64,23 @@ export default class QrCodeScanner extends Component<Props> {
 
   render() {
     return (
-      // eslint-disable-next-line
+      <Fragment>
+        {this.renderLoadingIndicator()}
+        {this.renderScanner()}
+      </Fragment>
+    )
+  }
+
+  renderLoadingIndicator() {
+    const { theme } = this.props
+    const { loading } = this.state
+
+    return loading ? <Loading theme={theme} nobackground /> : null
```

Comment:
Added a `nobackground` attribute since the default grayish background (in Loading.scss) doesn't suit the Send modal.